bears/xml/XMLBear.py

```diff
@@ -24,10 +26,16 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
+            self.use_stderr = True
+            if self.stdout_output:  # only yield Result if stdout is not empty
+                return collections.ChainMap(
+                           self._process_issues(self.stderr_output, filename),
```

Comment:
It should be 4 spaces from the beginning of "return"

On Fri, Mar 18, 2016 at 2:53 PM, vivek425ster <notifications@github.com>
wrote:

> In bears/xml/XMLBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/255#discussion_r56629795>
> :
>
> > @@ -24,10 +26,16 @@ class XMLBear(LocalBear, Lint):
> >      gives_corrected = True
> >
> >      def process_output(self, output, filename, file):
> > -        if self.stdout_output:  # only yield Result if stdout is not empty
> > -            return self._process_corrected(self.stdout_output, filename, file)
> > -        if self.stderr_output:  # pragma: no cover
> > +        if self.stderr_output:
> > +            self.use_stderr = True
> > +            if self.stdout_output:  # only yield Result if stdout is not empty
> > +                return collections.ChainMap(
> > +                           self._process_issues(self.stderr_output, filename),
>
> @AbdealiJK <https://github.com/AbdealiJK> What's the issue in indentation
> ?
> Isn't it 4 spaces PEP8 doesn't give me any error.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/255/files/f0425f051fba52543fd2984750a8277b427e655a#r56629795>
>